### DISCUSSION TITLE
Should we restrict the quest for elevation guidepost ?

### DISCUSSION THREAD

**Discussion by ravenfeld** [Helium314/SCEE#573] [Ideas]
In France and Belgium, we only have elevations if there is a name or if there isn't, it's very, very rare. 
At present, the quest does not contain the `and name`
This makes it possible to indicate that if there is a name to avoid asking when it cannot be indicated and cannot be resolved by saying that there is nothing.
Is it the same for you?
 If so, the rule I've put on my phone, I can offer it in PR.

**Comment by mcliquid** [comment]
In my region there is always a `ref` and an `ele`. A `name` only if the location has a specific name. On the German side of my neighborhood, there is almost never a `name`, always a `ref` and sometimes an `ele`. Therefore, in my neighborhood, this customization would not bring any added value.
But I would suggest two things in relation to SCEE: If the quest doesn't suit you, you can always customize it to your circumstances via the quest settings. Or you can deactivate the quests that don't make sense for you.

**Reply by ravenfeld** [reply]
Ok thanks for the info, so I'll just share on the French forum what they need to change so that they're not always asked to indicate the elevation when it's not possible.
